playbooks/roles/dev-patcher/tasks/main.yml

```diff
@@ -29,7 +33,7 @@
 
 - name: Get patch details
   include_tasks: patch-repos.yml
-  loop: "{{ dev_patcher_patches | default([], True) }}"
+  loop: "{{ dev_patcher_user_patches + dev_patcher_patches | default([], True) }}"
```

Comment:
Can just replace ```loop: "{{ dev_patcher_patches | default([], True) }}"``` with:

```
loop: "{{ (dev_patcher_patches) + (dev_patcher_user_patches | default([])) }}"
```

dev_patcher_patches: should always be defined in defaults/ so no need to default it. And The uter can just be defaulting to a list.